auth-web/src/store/modules/org.ts

```diff
@@ -314,6 +314,11 @@ export default class OrgModule extends VuexModule {
     this.currentOrganization = { ...this.currentOrganization, ...orgBusinessType }
   }
 
+  @Mutation
+  public setCurrentOrganizationBcolProfile (bcolProfile: BcolProfile) {
+    this.currentOrganization = { ...this.currentOrganization, bcolProfile }
```

Comment:
should this be `...bcolProfile` ?

Reply:
No, ...bcolProfile would spread `password` and `username` to the organization - `organization.password` `organization.username` which wouldn't be right